extensions/airship/airship-event-extension.schema.json

```diff
@@ -0,0 +1,271 @@
+          "title": "Triggering Push",
+          "description": "The specific push notification details associated with an event. An associated push helps you trace an event to the original notification or operation.An associated push object may specify a `time`, if the push was a singular operation sent at a defined time. Otherwise, the object will include a `group_id` if the push was sent at a relative time (`best_time` or `local_time`) an automation pipeline, or another operation resulting in multiple `push_id`s.",
+          "type": "object",
+          "properties": {
+            "https://ns.airship.com/campaign": {
+              "title": "Airship Campaigns",
+              "description": "An object listing the campaigns a push specification is associated with. The campaigns object includes an array of categories that must have between 1 and 10 elements, each of which is a string with a 64-byte limit.",
+              "type": "object",
+              "properties": {
+                "https://ns.airship.com/campaigns": {
+                  "title": "Airship Campaigns",
+                  "description": "An object listing the campaigns a push specification is associated with.",
+                  "type": "object",
+                  "isRequired": false,
+                  "properties": {
+                    "https://ns.airship.com/categories": {
+                      "title": "Airship Campaign Categories",
+                      "description": "An array of categories that must have between 1 and 10 elements, each of which is a string with a 64-byte limit.",
+                      "type": "array",
+                      "minItems": 1,
+                      "maxItems": 10,
+                      "items": {
+                        "type": "string",
+                        "maxLength": 64
+                      }
+                    }
+                  }
+                }
+              }
+            },
+            "https://ns.airship.com/push-id": {
+              "title": "Push ID",
+              "description": "A unique identifier for a push operation.",
+              "type": "string"
+            },
+            "https://ns.airship.com/group-id": {
+              "title": "Airship Group ID",
+              "description": "Identifies a push delivered over an interval of time, e.g. multiple push_ids as part of the fulfillment of an automation or a push-to-local-time specification.",
+              "type": "string"
+            },
+            "https://ns.airship.com/variant-id": {
+              "title": "Airship Variant ID",
+              "description": "The ID of the variant that a push is associated with, if the push was a part of an A/B test (experiment).",
+              "type": "string"
+            },
+            "https://ns.airship.com/push-time": {
+              "title": "Push Time",
+              "description": "The UTC time when the push occurred.",
+              "type": "date-time"
```

Comment:
This is not valid json-schema. The "type" you have here is actual the "format". The type should be "string". Please change to:
"type": "string",
"format": "date-time"